statuslight_control/lib/statuslight_control/statuslight.ex

```diff
@@ -1,26 +1,38 @@
 defmodule StatuslightControl.Statuslight do
   use GenServer
+  @renderer Application.get_env(:statuslight_control, :renderer)
+  @pixel_width 12
+  @pixel_height 1
 
-  def start_link(renderer) do
-    GenServer.start_link(__MODULE__, renderer, name: StatusLight)
+  def start_link(_) do
+    GenServer.start_link(__MODULE__, %{color: "green"}, name: __MODULE__)
```

Comment:
Instead of creating an ets table and storing color state there, we're storing it in the state of the main application GenServer.

Reply:
Makes sense.  I'm going to have a comment on the UI side of this around how relying on this couples statusmotron_ui to statuslight_control more tightly, but for now that is a moot point as they are already too coupled since I was rushing and didn't pay enough attention to that.  TL;DR - this is good with me but I might re-address how the UI consumes this later on if I get motivated to decouple them.